src/client/automation/playback/press/shortcuts.js

```diff
@@ -38,19 +38,19 @@ function updateTextAreaIndent (element) {
 
 function getLineIndentInTextarea (textarea) {
     var inverseSelection = textSelection.hasInverseSelection(textarea);
+    var cursorPosition   = inverseSelection
+        ? textSelection.getSelectionStart(textarea)
+        : textSelection.getSelectionEnd(textarea);
+    var textareaValue    = nativeMethods.textAreaValueGetter.call(textarea);
```

Comment:
We usually use this variant
```js
var cursorPosition   = inverseSelection ?
        textSelection.getSelectionStart(textarea) : 
        textSelection.getSelectionEnd(textarea);
```

Reply:
Let's use everywhere `domUtils.setElementValue` and `domUtils.getElementValue` instead of e.g. `nativeMethods.textAreaValueGetter.call(...)`.
What do use think?

Reply:
I think it is faster than the `getElementValue` or `setElementValue` functions because it runs without the `isTextAreaElement` and `isInputElement` conditions.